config/locales/en.yml

```diff
@@ -226,6 +229,12 @@ en:
               invalid: "You must enter a valid graduation year"
             country:
               blank: "You must select the country where the degree was obtained"
+        provider:
+          attributes:
+            name:
+              blank: You must provide a name
+            dttp_id:
+              invalid: You must provide a valid DTTP ID
```

Comment:
Can we have an example like 

https://github.com/DFE-Digital/register-trainee-teachers/blob/2a00b3e2c392f64d8a93774751b3b09feba75bb9/config/locales/en.yml#L244